flow-typed/npm/redux_v3.x.x.js

```diff
@@ -1,62 +1,109 @@
-// flow-typed signature: ba132c96664f1a05288f3eb2272a3c35
-// flow-typed version: c4bbd91cfc/redux_v3.x.x/flow_>=v0.33.x
+// flow-typed signature: 86993bd000012d3e1ef10d757d16952d
+// flow-typed version: a165222d28/redux_v3.x.x/flow_>=v0.33.x
 
 declare module 'redux' {
 
   /*
 
     S = State
     A = Action
-    R = Result of a thunk action
+    D = Dispatch
```

Comment:
I think the R comment is still useful here to retain.

Reply:
Ok, I'll add it back to types/store.js